packages/paste-core/components/form/src/select/Select.tsx

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import css from '@styled-system/css';
+import ChevronDown from '@twilio-paste/icons/svg/ChevronDown.svg';
+import {SelectInputType} from './types';
+import {Prefix} from '../shared/Prefix';
+import {Suffix} from '../shared/Suffix';
+import {FieldWrapper} from '../shared/FieldWrapper';
+import {safelySpreadFormControlProps} from '../shared/Utils';
+
+export interface SelectProps extends React.SelectHTMLAttributes<HTMLSelectElement> {
+  id: string;
+  className?: never;
+  disabled?: boolean;
+  required?: boolean;
+  value?: string; // returns empty string by default if no value is selected
+  type?: SelectInputType;
+  helpText?: string; // not sure if this one is needed...
```

Comment:
I don't think we need the `helpText` here. We can use `FormHelpText`, which it looks like you're doing in the stories.

Reply:
Ah yes, that makes sense. I think I need to clean up some of these other props too. We chatted briefly about this in the doc I shared with some questions, but are there any others here I can omit from the interface? I'm thinking: `type`, `options`, and `label`?

Reply:
@SiTaggart Any input here as well? ☝️

Reply:
I think we can drop `type` and `label`.